targets/curie_bsp/README.md

```diff
@@ -0,0 +1,120 @@
+###About Curie BSP port
```

Comment:
Because other `README.md`s in targets folders are started with level 3. I followed their format. Do you think we should change to level1?